datasetinsights/stats/coco_stats.py

```diff
@@ -0,0 +1,146 @@
+from tqdm import tqdm
+import numpy as np
+import math
+import pandas as pd
+import random
+
+
+COCO_AREA_RANGES = [
+    [0 ** 2, 1e5 ** 2],  # all
+    [0 ** 2, 32 ** 2],  # small
+    [32 ** 2, 96 ** 2],  # medium
+    [96 ** 2, 1e5 ** 2],  # large
+    [96 ** 2, 128 ** 2],  # 96-128
+    [128 ** 2, 256 ** 2],  # 128-256
+    [256 ** 2, 512 ** 2],  # 256-512
+    [512 ** 2, 1e5 ** 2]]  # 512-inf
+
+COCO_AREA_DICT = {
+        0: "all",
+        1: "small",
+        2: "medium",
+        3: "large",
+        4: "96-128",
+        5: "128-256",
+        6: "256-512",
+        7: "512-inf"}
+
+
+def _get_bbox_heatmap(coco_img_ids, coco_obj):
+    # calculate bbox shape according to largest image in the dataset
+    max_height, max_width = 0, 0
+    for idx in coco_img_ids:
+        img = coco_obj.loadImgs(idx)[0]
+        max_height = max(img['height'], max_height)
+        max_width = max(img['width'], max_width)
+
+    bbox_heatmap = np.zeros([max_height, max_width, 1])
+    return bbox_heatmap
+
+
+def _get_kpt_dict(coco_obj):
+    kpt_dict = {}
+    coco_kpts = coco_obj.loadCats(coco_obj.getCatIds())[0]['keypoints']
+    for key in coco_kpts:
+        kpt_dict[key] = 0
+    return kpt_dict
+
+
+def _get_bbox_dict():
+    bbox_dict = {}
+    for key in COCO_AREA_DICT:
+        bbox_dict[key] = 0
+    return bbox_dict
+
+
+def get_stats(img_ids, cat_ids, coco_obj):
```

Comment:
This method return a tuple of 4 objects. Can you break it down into two methods? four methods?